kubernetes/cray-postgresql/templates/_helpers.tpl

```diff
@@ -0,0 +1,91 @@
+*/}}
+{{- define "cray-postgresql.app-version" -}}
+{{- default "latest" .Values.global.appVersion | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "cray-postgresql.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "cray-postgresql.apiname"}}
+{{- printf "%s" .Chart.Name | replace "cray-" "" | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create base chart name and version as used by the chart label.
+*/}}
+{{- define "cray-postgresql.base-chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" | trimSuffix "_" | trimSuffix "." -}}
+{{- end -}}
+
+{{/*
+Create actual chart name and version as used by the chart label.
+*/}}
+{{- define "cray-postgresql.chart" -}}
+{{- printf "%s-%s" (.Values.global.chart.name | default "unknown-chart-name") (.Values.global.chart.version | default "unknown-chart-version") | replace "+" "_" | trunc 63 | trimSuffix "-" | trimSuffix "_" | trimSuffix "." -}}
+{{- end -}}
+
+{{/*
+Convenient way to refer to the metallb address pool inside or outside a partition
+*/}}
+{{- define "cray-postgresql.metallb-address-pool" -}}
+{{- if .Values.partitionId -}}
+{{- printf "%s-%s" .Values.service.network (.Values.partitionId | nospace | trunc 8) -}}
+{{- else -}}
+{{- printf .Values.service.network -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Convenient way to refer to the boot-services shared IP inside or outside a partition
+*/}}
+{{- define "cray-postgresql.boot-services-shared-ip" -}}
+{{- if .Values.partitionId -}}
+{{- printf "%s-%s" "boot-services" (.Values.partitionId | trunc 8) -}}
+{{- else -}}
+{{- printf "boot-services" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified name for postgresql DB Backup based on fullname.
+*/}}
+{{- define "cray-postgresql.postgresqlDbBackupFullname" -}}
+{{- printf "%s-postgresql-db-backup" (include "cray-postgresql.fullname" .) -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use for the postgres DB Backup
+*/}}
+{{- define "cray-postgresql.postgresqlDbBackupServiceAccountName" -}}
+{{- if .Values.sqlCluster.backup.serviceAccount.create -}}
+    {{ default (include "cray-postgresql.postgresqlDbBackupFullname" .) .Values.sqlCluster.backup.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.sqlCluster.backup.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
```

Comment:
We should not need any of this - this is all the old postgres backup solution - lines 75-91

For upgrades - we'll also need some way to clean up the old postgres-db backup cronjob - but that can be another ticket

Reply:
I've removed this section and will and a jira to rm the old backups.